salesforce/my-conservation-life/force-app/main/default/lwc/controllers/__tests__/assetDefinitionsController.test.js

```diff
@@ -21,6 +21,42 @@ describe('assetDefinitions.findAssetTypes', () => {
     });
 });
 
+describe('assetDefinitions.findAssetPropTypes', () => {
+    const EXPECTED_ASSET_PROP_TYPES = [];
+    let fetch;
+
+    let URL = utils.URL + 'assetPropTypes'
+
+    beforeEach(() => {
+        fetch = createMockFetch(EXPECTED_ASSET_PROP_TYPES);
+        global.fetch = fetch;
+    });
+
+    it('finds all the asset properties for a given asset type', async () => {
+        const assetPropTypes = await assetDefinitions.fetchAssetPropTypes(1);
+        expect(fetch.mock.calls[0][0]).toBe(URL);
+        expect(assetPropTypes).toEqual(EXPECTED_ASSET_PROP_TYPES);
+    });
+});
+
+describe('assetDefinitions.findAssetPropsByTypeID', () => {
+    const EXPECTED_ASSET_PROPERTIES = [];
+    let fetch;
+
+    let URL = utils.URL + 'assetPropsByTypeID';
+
+    beforeEach(() => {
+        fetch = createMockFetch(EXPECTED_ASSET_PROPERTIES);
+        global.fetch = fetch;
+    });
+
+    it('finds all the asset properties for a given type of asset', async () => {
+        const assetPropsByTypeID = await assetDefinitions.fetchAssetPropsByTypeID(1);
+        expect(fetch.mock.calls[0][0]).toBe(URL);
```

Comment:
check if the "1" is in the body. i.e. if the 2nd parameter in the mock fetch call is {assetTypeID: 1}